db/sql/0_msar.sql

```diff
@@ -101,10 +101,22 @@ $$ LANGUAGE plpgsql RETURNS NULL ON NULL INPUT;
 ----------------------------------------------------------------------------------------------------
 -- INFO FUNCTIONS
 --
--- Functions in this section get information about a given table or column.
+-- Functions in this section get information about a given schema, table or column.
 ----------------------------------------------------------------------------------------------------
 ----------------------------------------------------------------------------------------------------
 
+CREATE OR REPLACE FUNCTION __msar.get_sch_name(sch_id oid) RETURNS TEXT AS $$/*
+Return the name for a given schema, quoted as appropriate.
+
+Args:
+  sch_id: The OID of the schema.
+*/
+BEGIN
+	RETURN sch_id::regnamespace::text;
```

Comment:
Please use 2 spaces rather than a tab character for indentation to keep it consistent with the rest of the SQL code.

Reply:
For some reason, GitHub is detecting 2 space tabs as 4. I'll refactor to use spaces instead.